nvi-commons/src/test/resources/errorfreeModel.nt

```diff
@@ -0,0 +1,49 @@
+<https://api.sandbox.nva.aws.unit.no/cristin/person/1215176> <https://nva.sikt.no/ontology/publication#affiliation> <https://api.sandbox.nva.aws.unit.no/cristin/organization/194.0.0.0> .
```

Comment:
This is used in all of the NVI tests and is designed to pass all of the tests; I remove/add/replace data from it to test the logging.